packages/webapp/pages/reset-password.tsx

```diff
@@ -1,28 +1,97 @@
-import React, { ReactElement } from 'react';
+import React, { FormEvent, ReactElement, useState } from 'react';
```

Comment:
General question, can we not abstract the reset password component to work both here and in the UI?
Or atleast the function?

Reply:
I haven't checked yet how the reset password would look like. I will check how it would be processed then see how we can uniform their calls. Thanks for the mention.